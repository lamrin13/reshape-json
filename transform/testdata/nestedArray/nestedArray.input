{
    "name": "John",
    "street": "Dongg Street",
    "unit": "12",
    "role": "SWE",
    "last4": "1234",
    "expiryMonth": 12,
    "expiryYear": 2023,
    "fruits": [
        {
            "t1": "John",
            "t2": "john.a@abc.com"
        },
        {
            "t1": "Bob",
            "t2": "bob.b@abc.com"
        }
    ]
}