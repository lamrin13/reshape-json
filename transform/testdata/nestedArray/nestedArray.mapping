{
		"user.name":          "name",
		"user.role":          "role",
		"card.lastFour":      "last4",
		"card.expiry":        "expiryMonth+expiryYear/-",
		"address":            "unit+street",
		"user.favoriteFoods": "fruits",
		"o1":                 "t1",
		"o2":                 "t2"
	}