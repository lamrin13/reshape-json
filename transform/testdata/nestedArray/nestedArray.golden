{
    "user": {
        "name": "John",
        "role": "SWE",
        "favoriteFoods": [
            {
                "o1": "John",
                "o2": "john.a@abc.com"
            },
            {
                "o1": "Bob",
                "o2": "bob.b@abc.com"
            }
        ]
    },
    "card": {
        "lastFour": "1234",
        "expiry": "12-2023"
    },
    "address": "12 Dongg Street"
}